firmware/integration/rusefi_config.txt

```diff
@@ -22,22 +22,21 @@
 ! each field is declared as
 ! type name;comment
 
-
-#define TS_SIGNATURE "rusEFI v1.2020.3"
+! WHEN SHOULD I UPDATE THE BELOW VERSIONS?
+! - Update TS_SIGNATURE whenever either the flash format or TS output format changes (format: Year.Month.Sequential)
+! - Update RUSEFI_FIRMWARE_VERSION periodically as the firmware is updated.  Its value doesn't necessarily indicate dangerous
+!       incompatibility, but could indicate missing features.
+! - Update RUSEFI_FLASH_VERSION whenever the flash format is changed in a way that isn't backward compatible in a safe way (format: YearMonthDay)
+#define TS_SIGNATURE "rusEFI v2020.3.0"
+#define RUSEFI_FIRMWARE_VERSION 20200323
```

Comment:
at the moment in order to increase runtime version I modify one .cpp file, only one .cpp file needs to be recompiled

if we move runtime version to rusefi_config.txt - I would need to invoke generator and I will have one or multiple headers change which would require complete recompile. I would also have to push multiple files to git not just one file

this significantly increases the number of manual clicks for me to do. at this point I am not convinces that there is enough value in this change to justify the forever discomfort.